tests/integration/test_pwupdate.py

```diff
@@ -0,0 +1,175 @@
+"""Testing password update via API request
+"""
+import time
+from datetime import datetime, timedelta
+
+from . import BaseIntegrationTest
+from datameta.api import base_url
+from .utils import create_pwtoken
+
+
+class TestPasswordUpdate(BaseIntegrationTest):
```

Comment:
Can you look into using `parameterized` here, too?

Reply:
6/8 tests 'parameterized'